api/specs.proto

```diff
@@ -84,7 +84,7 @@ message ServiceSpec {
 		// Aliases specifies a list of discoverable alternate names for the service on this Target.
 		repeated string aliases = 2;
 	}
-	repeated NetworkAttachmentConfig networks = 7;
```

Comment:
@mrjana I think you misunderstood me: the inner type `NetworkAttachmentConfig` is no longer needed. `NetworkAttachmentConfig` is structurally identical.